crates/fluvio-extension-common/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-extension-common"
-version = "0.14.4"
+version = "0.14.5"
```

Comment:
Shouldn't this be bump to `0.15.0` since type is renamed?

Reply:
I added the aliais